packages/docs/pages/reference/elements/subtitle.mdx

```diff
@@ -8,12 +8,22 @@ import {
 
 # `<subtitle>`
 
-Short description of what the `<subtitle>` tag does.
+The `<subtitle>` tag can be used only in the narrow context to provide a
+subtitle to a book's or ariticle's main title.
 
 ## Syntax
 
 <SyntaxDisplay name="subtitle" />
 
 ## Examples
 
+```ptx-block
+<FRAGMENT>
+<article>
+    <title>Article Title</title>
+    <subtitle>Perhaps a subtitle</subtitle>
+    ...
```

Comment:
This is not valid pretext is it? There needs to be a `<p>...</p>` there. Though since this isn't a `ptx-example`, I suppose it could be okay...